code/game/gamemodes/miniantags/abduction/abduction_gear.dm

```diff
@@ -411,20 +411,28 @@ Congratulations! You are now trained for xenobiology research!"}
 	return 1
 
 /obj/item/weapon/abductor_baton/attack(mob/target, mob/living/user)
-	if(!IsAbductor(user))
+	if (!IsAbductor(user))
 		return
 
-	if(isrobot(target))
+	if (isrobot(target))
 		..()
 		return
 
-	if(!isliving(target))
+	if (!isliving(target))
 		return
 
 	var/mob/living/L = target
 
 	user.do_attack_animation(L)
-	switch(mode)
+
+	// [tgstation/#14599] Allow reactive teleport armor to kick in when attacked.
```

Comment:
Note that I've added this line to point to the bug that was fixed. I did this so that if someone has any problems with this addition in the future they'll know why it was added and be informed enough to assess if/how they should change it.

I just copied it from the stun baton code to quickly address this bug, but ideally we'd want to avoid copy/pasting code like this and work toward generalizing these kinds of things. Is there an object that shouldn't trigger a mob's defenses? If there isn't, it may be possible to resolve this in the calling proc.